Formula/singularity.rb

```diff
@@ -1,31 +1,46 @@
 class Singularity < Formula
-  desc "Application containers for Linux"
+  desc "Application container and unprivileged sandbox platform for Linux"
   homepage "https://singularity.hpcng.org"
   url "https://github.com/hpcng/singularity/releases/download/v3.8.2/singularity-3.8.2.tar.gz"
   sha256 "996611dec402b4d372b8b9456dd9ec1cb43712d08502e455f38521bd199856d3"
   license "BSD-3-Clause"
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, x86_64_linux: "565338bfa4b430b4e3de41de908e91b225e98fdcba64db857f2aedfc1c782be2"
   end
 
+  # No relocation, the localstatedir to find configs etc is compiled into the program
+  pour_bottle? only_if: :default_prefix
+
   depends_on "go" => :build
   depends_on "openssl@1.1" => :build
-  depends_on "libarchive"
+  depends_on "pkg-config" => :build
+  depends_on "libseccomp"
   depends_on :linux
-  depends_on "pkg-config"
   depends_on "squashfs"
-  depends_on "util-linux" # for libuuid
 
   def install
-    system "./mconfig", "--prefix=#{prefix}"
+    inreplace "pkg/util/singularityconf/config.go" do |s|
+      unsquashfs_dir = Formula["squashfs"].bin.to_s
+      s.sub!(/(directive:"mksquashfs path)/, "default:\"#{unsquashfs_dir}/bin\" \\1")
+
+      # When a cryptsetup formua added to brew, enable
+      # cryptsetup_dir = "#{Formula["cryptsetup"].bin}"
+      # s.sub! /(directive:"cryptsetup path)/, "default:\"#{cryptsetup_dir}/bin\" \\1"
```

Comment:
Please make a pull request for such a formula then.

Reply:
Ok, lets reach an approval for the other relocation point first. crypted container is a very optional feaure,
so I'd remove them for now. First priority is to get the package installed and working correctly.

Reply:
I removed the comment, it won't be needed.